Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "mozrunner"
-version = "0.3.4"
+version = "0.3.5"
```

Comment:
I think this should be version 0.4 because it changes behaviour (although not API) in a backwards incompatible way.